examples/tests/AuctionSpec.hs

```diff
@@ -68,38 +68,41 @@ hammerToWithdraw = do
 
 noBids :: MonadBlockChain m => m ()
 noBids = do
-  t0 <- currentTime
+  (t0, _) <- currentTime
   let deadline = t0 + 60_000
   offerOref <- A.txOffer (wallet 1) (banana 2) 30_000_000
   A.txSetDeadline (wallet 1) offerOref deadline
-  awaitTime (deadline + 1)
+  slotAfterDeadline <- getEnclosingSlot deadline <&> (+ 1)
+  awaitSlot slotAfterDeadline
   A.txHammer (wallet 1) offerOref
   return ()
 
 oneBid :: MonadBlockChain m => m ()
 oneBid = do
-  t0 <- currentTime
+  (t0, _) <- currentTime
   let deadline = t0 + 60_000
   offerOref <- A.txOffer (wallet 1) (banana 2) 30_000_000
   A.txSetDeadline (wallet 1) offerOref deadline
   A.txBid (wallet 2) offerOref 30_000_000
-  awaitTime (deadline + 1)
+  slotAfterDeadline <- getEnclosingSlot deadline <&> (+ 1)
+  awaitSlot slotAfterDeadline
   A.txHammer (wallet 3) offerOref -- It doesn't matter which wallet hammers here
 
 twoBids :: MonadBlockChain m => m ()
 twoBids = do
-  t0 <- currentTime
+  (t0, _) <- currentTime
   let deadline = t0 + 60_000
   offerOref <- A.txOffer (wallet 1) (banana 2) 30_000_000
   A.txSetDeadline (wallet 1) offerOref deadline
   A.txBid (wallet 2) offerOref 30_000_000
   A.txBid (wallet 3) offerOref 40_000_000
-  awaitTime (deadline + 1)
+  slotAfterDeadline <- getEnclosingSlot deadline <&> (+ 1)
+  awaitSlot slotAfterDeadline
```

Comment:
Since this is an example, and `<&>` is arguably not as well-known in the Haskell ecosystem as `<$>`, what do you think about
```suggestion
  deadlineSlot <- getEnclosingSlot deadline
  awaitSlot $ deadlineSlot + 1
```
here and below?